.github/workflows/test.yml

```diff
@@ -6,6 +6,10 @@ on:
   pull_request:
     branches: [main]
 
+concurrency:
```

Comment:
the tests were fast enough that i thought it would be better to see all errors as opposed to just (potentially) the first one. the primary failure is typically windows, so it probably doesn't matter much though.

Reply:
Just re-reading this and I think the behavior you're describing is handled by [`jobs.strategy.fail-fast`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast) below.

The [`concurrency`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast) setting here applies at the workflow level, so in this case it will only cancel an old workflow for a given branch if a new commit is pushed before the old one finishes.